pandas/tests/frame/methods/test_to_numpy.py

```diff
@@ -19,7 +19,8 @@ def test_to_numpy_dtype(self):
 
     def test_to_numpy_copy(self):
         arr = np.random.randn(4, 3)
-        df = DataFrame(arr)
+        with option_context("mode.data_manager", "block"):
```

Comment:
would @td.skip_array_manager_invalid_test make more sense here?

Reply:
> would @td.skip_array_manager_invalid_test make more sense here?

Indeed, will update (I think this was from before I added the decorators)